src/components/enterprise-user-subsidy/enterprise-offers/data/constants.js

```diff
@@ -30,14 +30,14 @@ export const NO_BALANCE_CONTACT_ADMIN_TEXT = 'Contact administrator';
 
 export const OFFER_BALANCE_CLICK_EVENT = 'edx.ui.enterprise.learner_portal.offer_balance_alert.clicked';
 
+// TODO: why are these Learner Credit policies and content assignments related constants in this file?
```

Comment:
[inform] TODO comment is intentional/deferred for later. These constants do not belong in a module related to `enterprise-offers`.